frontend/lib/src/components/elements/Popover/Popover.tsx

```diff
@@ -127,16 +135,13 @@ const Popover: React.FC<React.PropsWithChildren<PopoverProps>> = ({
         {/* This needs to be wrapped into a div, otherwise
         the BaseWeb popover implementation will not work correctly. */}
         <div>
-          <BaseButtonTooltip
-            help={element.help}
-            containerWidth={element.useContainerWidth}
-          >
+          <BaseButtonTooltip help={element.help} containerWidth={true}>
             <BaseButton
               data-testid="stPopoverButton"
               kind={BaseButtonKind.SECONDARY}
               size={BaseButtonSize.SMALL}
               disabled={empty || element.disabled}
-              containerWidth={element.useContainerWidth}
+              containerWidth={true}
```

Comment:
nit: `containerWidth` is probably something that can be removed out at some point?